packages/awkward-cpp/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: awkward-cpp
+  version: 22
+  top-level:
+    - awkward_cpp
+
+source:
+  url: https://pypi.io/packages/source/a/awkward-cpp/awkward-cpp-22.tar.gz
+  sha256: 21679636fb21cfe3715f88a32326a579199384db2da4a62995c310502d7fe85f
+
+build:
+  script: |
+    export CMAKE_ARGS="${CMAKE_ARGS} -DEMSCRIPTEN=1"
+  exports: whole_archive
```

Comment:
What happens if you don't set `whole_archive`?

Reply:
It used to be the case that some symbols were missing. Right now I can't seem to test this locally; I'm getting 
```
OSError: [Errno 7] Argument list too long: 'em++'
```

I don't recall this happening before.

Reply:
I think it's okay for now then then, we have a bug in our symbol logic that causes this `Argument list too long`  error when there are a lot of exported symbols.